### PR TITLE
4.3.0 changelog

### PR DESCRIPTION
**Pull request by mx-psi**
## What does this PR do?

- Create 4.3.0 changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+# 4.3.0 / 2020-07-06
+
+* [FEATURE] Record installation information for telemetry and troubleshooting purposes. See [#281][].
+* [BUGFIX] Fix error when facts value doesn't exist on Redhat OS family of the arm architecture. See [#283][]. Thanks to [@kanga333][].
+* [BUGFIX] Fix idempotence when reinstalling same pinned version See [#269][].
+* [DOCS] Update wording on blocking install of problematic agent versions. See [#284][].
```
Comment (albertvaka) [inline CHANGELOG.md]:
```suggestion
```
Docs are not something we "release", so we can remove it from the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -219,9 +226,13 @@ Initial release, compatible with Ansible v1 & v2
 [#264]: https://github.com/DataDog/ansible-datadog/issues/264
 [#267]: https://github.com/DataDog/ansible-datadog/issues/267
 [#268]: https://github.com/DataDog/ansible-datadog/issues/268
+[#269]: https://github.com/DataDog/ansible-datadog/issues/269
 [#270]: https://github.com/DataDog/ansible-datadog/issues/270
 [#271]: https://github.com/DataDog/ansible-datadog/issues/271
 [#275]: https://github.com/DataDog/ansible-datadog/issues/275
+[#281]: https://github.com/DataDog/ansible-datadog/issues/281
+[#283]: https://github.com/DataDog/ansible-datadog/issues/283
+[#284]: https://github.com/DataDog/ansible-datadog/issues/284
```
Comment (albertvaka) [inline CHANGELOG.md]:
```suggestion
```

**Review by albertvaka — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (albertvaka) [reply CHANGELOG.md]: And in general, we can skip changes that are this small and not user-facing.

Comment (mx-psi) [reply CHANGELOG.md]: There are some docs changes documented before, but I guess they were bigger/more relevant.

**Pull request merged**